app/drivers/zephyr/kscan_gpio_direct.c

```diff
@@ -33,9 +33,9 @@ struct kscan_gpio_config {
 };
 
 struct kscan_gpio_data {
-#if defined(CONFIG_ZMK_KSCAN_GPIO_POLLING)
+#if defined(ZMK_KSCAN_DIRECT_POLLING)
```

Comment:
I think the macros are all defined with CONFIG_ prefix.
Please test this with and without polling and make sure this works

Reply:
I think this might have been a merge issue before submitting. Will test and correct.